clientlib/src/main/proto/yelp/nrtsearch/search.proto

```diff
@@ -492,7 +493,7 @@ message QueryRescorer {
     Query rescoreQuery = 1;
     double queryWeight = 2;
     double rescoreQueryWeight = 3;
-
+    string name = 4;
```

Comment:
Maybe the name could be added to the `Rescorer` message. This would avoid collisions if the same plugin rescorer gets used multiple times.

Reply:
Fields has been moved to the `Rescorer` message